ci-operator/step-registry/vsphere/deprovision/usertags/vsphere-deprovision-usertags-commands.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+echo "$(date -u --rfc-3339=seconds) - sourcing context from vsphere_context.sh..."
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+
+printf '%s' "${USER_TAGS:-}" | while read -r tag
+do
+    govc tags.rm -c machine-cate ${tag}
```

Comment:
If there are two concurrent job running at the same time, while the removal from one of two jobs will cause conflicts?

Reply:
good catch. i will update code to check if tags exist first.

Reply:
I guess you can add prefix for tag category instead of hardcode as "machine-cate", e.g: 
CLUSTER_NAME="${NAMESPACE}-${UNIQUE_HASH}"
tag_category_name="${CLUSTER_NAME}-usertags-category"

so that we can directly destroy it when testing is finished, and no need to consider conflict issue in this way.

Reply:
got it.